### PR TITLE
Allow signin with email

### PR DESCRIPTION
**Pull request by abulojoshua1**
### What does the PR do?
- Allows registered users to sign in using their email or username

### Changes made.
- Set an allauth settings flag to allow registered users to sign in using their email or username

### CODE REVIEW

**Comment by abulojoshua1** [comment]
> Please create a tests module inside /people folder and create a test for username / email signin.

Ok, I'll create tests for this.

File: ihp/people/tests/test_login.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+import pytest
+from django.test import TestCase
+from django.urls import reverse
+
+from ihp.people.models import IHPProfile
+
+
+@pytest.mark.django_db
+class TestUserLogin(TestCase):
```
Comment (petrus7) [inline ihp/people/tests/test_login.py]: All of them tests allauth code. is it necessary?

Comment (abulojoshua1) [reply ihp/people/tests/test_login.py]: :) Yes, all those tests are testing `allauth` code (which I believe is already tested by the alluath team). But I do not think it unnecessary to add an extra layer of protection whenever possible.

Comment (abulojoshua1) [reply ihp/people/tests/test_login.py]: ^ Kindly let me know what you think.

**Pull request merged**